doc/api/corepack.md

```diff
@@ -64,6 +64,12 @@ package manager version you wish to set:
 corepack prepare yarn@x.y.z --activate
 ```
 
+Alternately, a [`dist-tag`](https://docs.npmjs.com/downloading-and-installing-packages-locally#installing-a-package-with-dist-tags) may be used:
```

Comment:
Can you use a [collapsed reference link](https://spec.commonmark.org/0.30/#collapsed-reference-link) to please the linter please?
```suggestion
Alternately, a [`dist-tag`][] may be used:
```